src/index.html

```diff
@@ -1,23 +1,43 @@
 <html>
   <head>
-   
+    <title>Turbulenz POC</title>
+    <link rel="stylesheet" type="text/css" href="css/main.css" />
   </head>
   <body>
-    <canvas id="mainCanvas" width="1024" height="768" />
+    <div id="main">
+      <div id="header">
+        <h1>
+          Turbulenz POC
+        <h1>
+        <div id="errorBox" class="error">

```

Comment:
Class error because errors are always going to be red on a yellow background for max hideousness.